data/pom.xml

```diff
@@ -159,26 +159,10 @@
             <groupId>com.azure</groupId>
             <artifactId>azure-core</artifactId>
         </dependency>
-        <!-- https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient -->
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>${httpclient.version}</version>
-            <exclusions>
-                <exclusion>
-                    <artifactId>httpcore</artifactId>
-                    <groupId>org.apache.httpcomponents</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>commons-codec</artifactId>
-                    <groupId>org.apache.commons</groupId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.httpcomponents</groupId>
-            <artifactId>httpcore</artifactId>
-            <version>${httpcore.version}</version>
+            <version>${apache.httpclient.version}</version>
```

Comment:
@AsafMah another reviewer asked if we could remove those lines in a different PR. Probably worth excluding them still. I will put those lines back.

Reply:
[Readded in this commit](https://github.com/Azure/azure-kusto-java/pull/335/commits/c25960feb5c5a0ba84b3744564aa7fb2a5b262a3)

Reply:
Reverted removing core in [this commit](https://github.com/Azure/azure-kusto-java/pull/335/commits/350dc1bb9ea59b11ba863c4f6c901ba04e55feb7) because the URIBuilder uses utilities out of core. Kept the exclusion of commons-codec though for better security posture.